src/usr/local/emhttp/plugins/tailscale/include/common.php

```
@@ -2,14 +2,28 @@
-foreach (glob(PLUGIN_ROOT . "/include/" . __NAMESPACE__ . "/*.php") ?: array() as $file) {
-    try {
-        require $file;
-    } catch (\Throwable $e) {
-        Utils::logmsg("Caught exception in {$file} : " . $e->getMessage());
-    }
-}
+// @phpstan-ignore requireOnce.fileNotFound
+require_once "/usr/local/php/unraid-tailscale-utils/vendor/autoload.php";
 
-Utils::setPHPDebug();
+$utils = new Utils(PLUGIN_NAME);
```

PHPStan (2.1.15)

Class Tailscale\Utils does not have a constructor and must be instantiated without any parameters.


(new.noConstructor)